library-core/build.gradle

```diff
@@ -41,45 +41,45 @@ android {
 
     // android supports & views dependencies
-    compile 'com.android.support:appcompat-v7:25.4.0'
-    compile 'com.android.support:design:25.4.0'
-    compile 'com.android.support.constraint:constraint-layout:1.0.2'
-    compile 'com.android.support:recyclerview-v7:25.4.0'
-    compile 'com.android.support:cardview-v7:25.4.0'
+    implementation "com.android.support:appcompat-v7:$androidSupportVersion"
+    implementation "com.android.support:design:$androidSupportVersion"
+    implementation "com.android.support.constraint:constraint-layout:1.0.2"
+    implementation "com.android.support:recyclerview-v7:$androidSupportVersion"
+    implementation "com.android.support:cardview-v7:$androidSupportVersion"
 
     //rxjava & rxandroid dependencies
-    compile "io.reactivex.rxjava2:rxjava:$rxjavaVersion"
-    compile "io.reactivex.rxjava2:rxandroid:$rxandroidVersion"
-    compile "com.jakewharton.rxbinding2:rxbinding:$rxbindingVersion"
+    implementation "io.reactivex.rxjava2:rxjava:$rxjavaVersion"
+    implementation "io.reactivex.rxjava2:rxandroid:$rxandroidVersion"
+    implementation "com.jakewharton.rxbinding2:rxbinding:$rxbindingVersion"
 
     //retrofit & networking dependencies
-    compile "com.squareup.retrofit2:converter-gson:$retrofitVersion"
-    compile "com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion"
-    testCompile "com.squareup.retrofit2:retrofit-mock:$retrofitVersion"
+    implementation "com.squareup.retrofit2:converter-gson:$retrofitVersion"
+    implementation "com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion"
+    testImplementation "com.squareup.retrofit2:retrofit-mock:$retrofitVersion"
 
     //jwt decoding library
     //see https://github.com/auth0/JWTDecode.Android
-    compile "com.auth0.android:jwtdecode:$jwtdecodeVersion"
+    implementation "com.auth0.android:jwtdecode:$jwtdecodeVersion"
 
     // location
-    compile 'com.google.android.gms:play-services-location:11.6.0'
-    compile 'com.google.android.gms:play-services-maps:11.6.0'
-    compile('com.mapbox.mapboxsdk:mapbox-android-sdk:5.1.2@aar') { transitive=true }
-    compile('com.mapbox.mapboxsdk:mapbox-android-services:2.2.0@aar') { transitive=true }
-    compile 'com.mapbox.mapboxsdk:mapbox-android-plugin-locationlayer:0.1.0'
+    implementation 'com.google.android.gms:play-services-location:11.6.0'
+    implementation 'com.google.android.gms:play-services-maps:11.6.0'
+    implementation('com.mapbox.mapboxsdk:mapbox-android-sdk:5.1.2@aar') { transitive=true }
+    implementation('com.mapbox.mapboxsdk:mapbox-android-services:2.2.0@aar') { transitive=true }
+    implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-locationlayer:0.1.0'
 
     // images
-    compile 'com.github.bumptech.glide:glide:4.3.1'
-    annotationProcessor 'com.github.bumptech.glide:compiler:4.3.1'
+    implementation 'com.github.bumptech.glide:glide:4.3.1'
+    annotationProcessor 'com.github.bumptech.glide:implementationr:4.3.1'
```

Comment:
Please take a second look at this. I suspect that your find and replace was a bit too inclusive, and that this should still be `compiler:4.3.1`

Reply:
Thanks Kristen I'm fixing it now